packages/atomic/src/components/common/facets/facet-value-label-highlight/facet-value-label-highlight.ts

```diff
@@ -0,0 +1,26 @@
+import {multiClassMap, tw} from '@/src/directives/multi-class-map';
```

Comment:
Must these changes be included in this PR?

We already have https://github.com/coveo/ui-kit/pull/5327 that covers this.